docs/book/diagnostics.md

```diff
@@ -451,7 +451,7 @@ Make sure all migrations are applied:
 
 ```php
 <?php
-use Doctrine\DBAL\Migrations\Configuration\Configuration;
+use Doctrine\Migrations\Configuration\Configuration;
```

Comment:
Is this namespace valid across both V1 and V2 releases? If not, we will need to provide an autoloader shim to perform class aliasing for compatibility purposes.

Reply:
@weierophinney I already did it. Check the file `src/autoload.php` in that PR:
https://github.com/zendframework/zend-diagnostics/pull/97/files#diff-e0ef674d7fb0e2a134b65bb8b1afba9c